app/services/api/v1/data/emission_pathways/filter.rb

```diff
@@ -34,7 +34,7 @@ def initialize(params)
           def call
             @query = apply_filters(@query, mview: true)
             @years_query = apply_filters(@years_query, mview: false)
-            @years = @years_query.distinct(:year).pluck(:year).sort
+            @years = @years_query.distinct(:year).pluck(:year).sort.reverse
```

Comment:
sure? certainly the opposite was requested two months ago 😕  https://www.pivotaltracker.com/story/show/159961025

Reply:
:(